00-HTML-CSS-basics/html/media.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>Media objects</title>
+		<meta name="viewport" content="width=device-width,initial-scale=1"/>
+		<meta name="author" content="Braulio Evangelisti"/>
+		<meta name="description" content="Media objects example">
+	</head>
+	<body>
+		<header>
+			<h1>Media objects</h1>
+		</header>
+		<section>
+			<h2>A list of media objects</h2>
+			<article>	
+				<h2>Image</h2>	
```

Comment:
Keep an eye on headings usage. Have in mind that these should be nested in your category heading (``<h2>A list of media objects</h2>``). Maybe *Image*, *Video* and *Audio* could start from `h3`.